src/GslCore/BasicMarkerProvider.fs

```diff
@@ -0,0 +1,99 @@
+﻿module BasicMarkerProvider
+open constants
+open LegacyParseTypes
+open commonTypes
+open Amyris.Dna
+open PluginTypes
+open DnaCreation
+
+/// Default marker provider if nothing else better
+let jobScorerBasicMarkerProvider _ = Some 0.0<PluginScore>
```

Comment:
Since this one-liner is only used in the provider below, please move the code right into the method declaration.